code/modules/surgery/organs/lavaland.dm

```diff
@@ -0,0 +1,340 @@
+/obj/item/organ/colossus
+	name = "Voice of God"
```

Comment:
change to `deus vocal cords` or something. Capital letters in item names are bad